### PR TITLE
Running the data retention job manually

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: doc/troubleshooting.md [hunk 1]
```diff
@@ -143,12 +143,49 @@ The `pg_restore` tool is used to restore a PostgreSQL database from an archive c
 pg_restore -h another.host.com -p 5432 -U postgres -d hoh hoh-$(date +%d-%m-%y_%H-%M).tar
 ```
 
-## Cronjob Compliance Data Restore
+## Cronjobs
```
Comment (ldpliu) [inline doc/troubleshooting.md:146]: Please also change the link and  description here. 
https://github.com/stolostron/multicluster-global-hub/pull/636/files#diff-a0842ea73f3a116eff7958fbf833dd2280c366c0d8985aa05f8e1403e0f39273R190
Thanks

Comment (yanmxa) [reply doc/troubleshooting.md:146]: Done!

File: doc/README.md [hunk 2]
```diff
@@ -187,7 +187,7 @@ This alert watch the [Cron jobs](#Cronjobs-and-Metrics) failed events. There are
   If this alert rule becomes firing, it means the [Local compliance status sync job](#Local-compliance-status-sync-job) failed. It may cause the data to be lost in the `history.local_compliance` table. Please check it and [manually run the job](./how_global_hub_works.md#running-the-summarization-process-manually)
 
 - Data Retention Job Failed
-  If this alert rule becomes firing, it means the [Partition job](#Partition-job) failed. 
+  If this alert rule becomes firing, it means the [Partition job](#Partition-job) failed and you can also [run it manually](./troubleshooting.md#run-the-data-retetion-job-manually).
```
Comment (ldpliu) [inline doc/README.md:190]: I think it's better to change the `Partition job` to `Data retention job` here?

Comment (yanmxa) [reply doc/README.md]: Done!

**Pull request merged**